troupon/static/scss/partials/_typography.scss

```diff
@@ -0,0 +1,47 @@
+
+/* ==========================================================================
+Typography
+========================================================================== */
+
+body {
+	font-family: font-family(primary);
+}
+
+@each $tag, $font-weight in (h1: light, h2: regular, h3: bold, h4: bold, body: regular) {
+	#{$tag} {
+		font: {
+			weight: font-weight(#{$font-weight});
+			size: font-size(#{$tag});
+		}
+		color: color(#{$tag});
+	}
+}
+
+h1 {
+	letter-spacing: -1px;
+}
+
+.mini-text {
+  font-size: font-size(mini);
+}
+
+
+.mini-label {
+	font: {
+		weight: font-weight(black);

```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Properties should be ordered size, weight